### PR TITLE
Fs jul19 yevhen sag - catalog solution

### PR DESCRIPTION
**Pull request by eugeneSahach**
- [DEMO LINK](https://yevhen-sahach.github.io/layout_catalog/)
- [TEST REPORT LINK](https://yevhen-sahach.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -10,10 +10,203 @@
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <header class="header">
+    <a href="#" class="logo"><img src="images/logo.png" alt="logo"></a>
```
Comment (irenhh) [inline src/index.html]: nested tags must be placed on the new line

File: src/index.html [hunk 2]
```diff
@@ -10,10 +10,203 @@
...
+        <li class="nav-item">
+          <a href="#" class="nav-link">видео</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="catalog">
+    <div class="product-card" data-qa="card">
+      <img src="images/imac.jpeg" alt="imac" class="product-image">
+      <p class="product-name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
```
Comment (irenhh) [inline src/index.html]: wrap this title in heading tag

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,161 @@
+body {
+  margin: 0;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
```
Comment (irenhh) [inline src/style.css]: it's 100% by default, it was mentioned when your previous task was reviewed. Please, remember to apply previous comments here too

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-link {
+  position: relative;
+  display: flex;
+  text-align: center;
+  font-family: Roboto;
```
Comment (irenhh) [inline src/style.css]: add extra font in case Roboto isn't loaded

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+  font-style: normal;
```
Comment (irenhh) [inline src/style.css]: remove font-style, it's default value

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+  box-sizing: border-box;
+  display: flex;
+  flex-flow: column nowrap;
+  width: 200px;
+  height: 403px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: Roboto;
+  font-weight: normal;
```
Comment (irenhh) [inline src/style.css]: no need to add font-weight here

File: src/style.css (hunk 4 above)
Comment (irenhh) [reply src/style.css]: you can specify font in body tag and delete in other places in your file

**Comment by yevhen-sahach** [comment]
Applied all sugestions from this PR and from previous PRs

**Pull request closed without merging**